core/sqf/src/seatrans/.gitignore

```diff
@@ -12,3 +12,6 @@ hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/Clean
 hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/KeyValueListScanner.java
 hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/MemoryUsageChore.java
 hbase-trx/src/main/java/org/apache/hadoop/hbase/regionserver/transactional/TrxTransactionState.java
+hbase-trx/src/main/java/org/apache/hadoop/hbase/client/ClientScanner98.java
```

Comment:
In 1.0 (actually after the fact I discovered that early 1.0 also didn't have the feature), and even in a latter version of 98, hbase introduced replica RS. This feature makes the client scanner multithread and not working with parallel scanner. So I froze an old version of non multi thread scanner, that does not support replica to be used with parallel scanner. That is ClientScanner98. It works with our 1.x HBase version.